Renegade X Launcher/ServerInfo.cs

```diff
@@ -458,47 +472,45 @@ public static string GetPrettyMapName(string map)
         {
             try
             {
-                PingReply reply = await new Ping().SendPingAsync(serverInfo.IPAddress);
+                PingReply reply = await new Ping().SendPingAsync(serverInfo.IpAddress);
+                Logger.Instance.Write($"Pinging server {serverInfo.IpWithPort} returned {reply.RoundtripTime}ms latency");
                 serverInfo.Ping = (int)reply.RoundtripTime;
-                if (serverInfo.PropertyChanged != null)
-                {
-                    serverInfo.PropertyChanged(serverInfo, new PropertyChangedEventArgs("Ping"));
-                }
+                serverInfo.PropertyChanged?.Invoke(serverInfo, new PropertyChangedEventArgs("Ping"));
             }
             catch (Exception e)
             {
-                Console.WriteLine("Failed to ping server: " + e.ToString());
+                Logger.Instance.Write($"Unable to ping server {serverInfo.IpWithPort} | {e.Message}\r\n{e.StackTrace}", Logger.ErrorLevel.ErrError);
             }
         }
 
         /// <summary>
         /// Tries to grab the countryname based on the given IPAddress.
         /// </summary>
-        /// <param name="IPAddress">The IPAddress</param>
+        /// <param name="ipAddress">The IPAddress</param>
         /// <returns>The name of the country that belongs to the IPAddress</returns>
-        static async Task<string[]> GrabCountry(string IPAddress)
+        static async Task<string[]> GrabCountry(string ipAddress)

```

Comment:
self note: I still need to add country codes to the server list so we can remove this from the launcher